### DISCUSSION TITLE
English version of the Harbor book

### DISCUSSION THREAD

**Discussion by masudkhan1** [goharbor/harbor#18157] [Polls]
Hi,

I recently started working with Harbor and need to know internal details in order to do my job.  An english version of the harbor book would be super helpful.  Please vote if you feel the same way!

Thanks!

**Comment by hainingzhang** [comment]
The English version of Harbor Book is available now. You can get a soft or hard copy via various online stores. Here are just a few links for your references:

https://www.amazon.com/Authoritative-Guide-Harbor-Management-Artifacts/dp/981192726X

https://link.springer.com/book/10.1007/978-981-19-2727-0
